src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -54,8 +56,12 @@
 public class ForgeRegistry<V> implements IForgeRegistryInternal<V>, IForgeRegistryModifiable<V>
 {
     public static Marker REGISTRIES = MarkerManager.getMarker("REGISTRIES");
-    private static Marker REGISTRYDUMP = MarkerManager.getMarker("REGISTRYDUMP");
-    private static Logger LOGGER = LogManager.getLogger();
+    private static final Marker REGISTRYDUMP = MarkerManager.getMarker("REGISTRYDUMP");
+    private static final Logger LOGGER = LogManager.getLogger();
```

Comment:
Try to avoid unnecessary changes like these in scoped PRs. Yes, they're good, but I think that if it isn't part of the PR, it should be saved for later.

Reply:
Bit nit-picky tbh... It's nearby and obvious, with such a small change not really warranting the effort of making a new branch, setting up Forge again, creating a PR, etc...

Reply:
Yeah no reason for you to change it back, that's why I mentioned it but still gave the thumbs up.